tips/technology-resources.md

```diff
@@ -222,28 +223,40 @@
 
 ## Javascript
 
-- [Arrayzing](https://gist.github.com/ourmaninamsterdam/1be9a5590c9cf4a0ab42#user-content-create-an-array) - Gives you answer on how to manage arrays by indexing topics from what you want to do, not the method names
+#### Books
```

Comment:
I think, in the context of an expanding section, sub-sections make sense here (especially for something as broad as JavaScript).